src/main/java/com/powsybl/openloadflow/ac/equations/AcEquationSystem.java

```diff
@@ -72,8 +65,7 @@ private static void createGeneratorControlEquations(LfBus bus, LfNetworkParamete
             if (voltageControl.isVoltageControlLocal()) {
                 createLocalVoltageControlEquation(bus, networkParameters, equationSystem, creationParameters);
             } else if (bus.isVoltageControlled()) {
-                // remote controlled: set voltage equation on this controlled bus
-                createVoltageControlledBusEquations(voltageControl, networkParameters, equationSystem, creationParameters);
+                createRemoteVoltageControlEquations(voltageControl, networkParameters, equationSystem, creationParameters);
```

Comment:
If I remember well we have also a case where we create distribution equations for remote and local controllers. We fall in that second case. You can keep `remote` in the name of the method, but maybe I can add it in the comment.

Reply:
Looking at the code there is nothing specific to local controller even if it works with hybride remote/local controller. The only local mode is handled in the previous if.